subsys/bluetooth/mesh/shell/shell_scheduler_cli.c

```diff
@@ -0,0 +1,155 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <stdlib.h>
+#include <bluetooth/mesh/models.h>
+#include <shell/shell.h>
+#include <sys/math_extras.h>
+
+#include "mesh/net.h"
+#include "mesh/access.h"
+#include "../model_utils.h"
+
+#include "shell_utils.h"
+
+static struct bt_mesh_model *mod;
+static struct bt_mesh_schedule_entry set_entry;
+
+static int cmd_get(const struct shell *shell, size_t argc, char *argv[])
+{
+	if (!mod && !shell_model_first_get(BT_MESH_MODEL_ID_SCHEDULER_CLI, &mod)) {
+		return -ENODEV;
+	}
+
+	struct bt_mesh_scheduler_cli *cli = mod->user_data;
+	uint16_t rsp;
+
+	int err = bt_mesh_scheduler_cli_get(cli, NULL, &rsp);
+
+	if (!err) {
+		shell_print(shell, "Status: %d", rsp);
+	}
+
+	return err;
+}
+
+static void schedule_print(const struct shell *shell, int err, struct bt_mesh_schedule_entry rsp)
+{
+	if (!err) {
+		shell_print(shell,
+			    "Date: %02d.%02d.%02d, Time: %d:%d:%d,\n"
```

Comment:
Entry cannot be used for date and time representation since there are a lot of specific predefined values that do not mean time or data explicitly. Look at `scheduler.h` for details

Reply:
I see, like ONCE_A_DAY etc. I dont see a good solution to this without extencive overhead in the print function, so how about changing the print format to just bluntly stating the param name and the value? The shell cli models is in its current state mostly suitable for people capable of reading the model spec, since most enumerated values etc. are passed and printed just like any other value, without extensive description of which values are valid or not. This could of course be made more user friendly, but it would require a fair amount of additional work.

Reply:
No simple solution for this. Otherwise, you will have to implement time conversion functionality that I spent an enormous amount of time during scheduler model implementation.
I'm Ok with stating param name and state. Probably only it is worth to add in brackets some logic meaning for special values if it doesn't cause significant overhead.